WordPress/Classes/ViewRelated/NUX/LoginViewController.m

```diff
@@ -177,7 +177,7 @@ - (void)bindToViewModel
 - (void)update3DTouchForLogIn
 {
     WP3DTouchShortcutCreator *shortcutCreator = [WP3DTouchShortcutCreator new];
-    [shortcutCreator createShortcuts:self.cancellable];
+    [shortcutCreator createShortcutsIf3DTouchAvailable:self.cancellable];
```

Comment:
I know it's not new code, but `LoginViewController.cancellable != loggedIn`. It is now, but if we start supporting a logged out experience, I'd guess logins would be cancellable even if logged out, and this would break. `WP3DTouchShortcutCreator` already checks if `hasWordPressComAccount`, why do we need the extra parameter?